xonsh/main.py

```diff
@@ -113,7 +113,7 @@ def path_argument(s):
                     metavar='args',
                     help='Additional arguments to the script specified '
                          'by script-file',
-                    nargs='*',
+                    nargs=argparse.REMAINDER,
```

Comment:
This says it better than I can in a PR :) http://unix.stackexchange.com/a/46856/129048